packages/core/content-manager/server/controllers/collection-types.js

```diff
@@ -329,6 +329,31 @@ module.exports = {
 
     const number = await entityManager.getNumberOfDraftRelations(id, model);
 
+    return {
+      data: number,
+    };
+  },
+  async getMultipleEntriesNumberOfDraftRelations(ctx) {
+    const { userAbility } = ctx.state;
+    const params = new URLSearchParams(ctx.querystring);
+    const ids = params.get('ids').split(',');
```

Comment:
you should be able to get this ids via ctx.params, and instead of sending them as a comma sepparated string, you could send them as an array

If I am not mistaken `ids=[1,2]` should be valid, if not if you can encode it with the qs library.

Reply:
this is what I get if I passed `ids=[1,2]` and I console log `ctx.params`

Reply:
let me know if the solution provided
```
const params = new URLSearchParams(ctx.querystring);
    const ids = JSON.parse(params.get('ids'));
```
it is ok for you, thanks

Reply:
?ids=[1,2] should be on ctx.request.query not params

Reply:
ok thanks, done